syntax = "proto3";

package user;

service UserService {
    rpc AddUser (UserRequest) returns (UserResponse);
    rpc GetUser (UserRequest) returns (UserResponse);
    rpc UpdateUser (UserRequest) returns (UserResponse);
    }

message UserRequest {
    string name = 1;
    string email = 2;
    int32 age = 3;
    }

message UserResponse {
    string message = 1;
    User user = 2;
    }

message User {
    int32 id = 1;
    string name = 2;
    string email = 3;
    int32 age = 4;
    }
